### DISCUSSION TITLE
Switching from Debian Stable to the Testing // Repositories

### DISCUSSION THREAD

**Discussion by AlphaElwedritsch** [SpiralLinux/SpiralLinux-project#381] [Q&A] [answered]
Hi
after switching to testing, do I have to enable the DFSG Repositories again? Or are these only relevant for stable?

once again thank you very much

![image](https://github.com/SpiralLinux/SpiralLinux-project/assets/109347938/21a5db63-2761-439d-bbfa-54b84542357c)

**Comment by geckolinux** [comment, marked as answer]
Hi there, Testing and Sid don't require any other repositories.

**Reply by AlphaElwedritsch** [reply]
Thank you
I've switched my personal Laptop now to Testing and gave my Mother an older one with stable.
She's happy. Browser, E-Mail and LibreOffice, that's all she need...

Conclusion: **SpiralLinux rocks** :+1:

**Comment by AlphaElwedritsch** [comment]
Is it a good idea to add:
`deb https://security.debian.org/debian-security testing-security main contrib non-free non-free-firmware
`to /etc/apt/sources.list, when using testing?

**Reply by geckolinux** [reply]
Interesting, I actually haven't heard of that one. I'll have to look into it a bit more.

**Reply by marwenius** [reply]
see "Security" under https://wiki.debian.org/Status/Testing

**Comment by AlphaElwedritsch** [comment]
@geckolinux could you get a look, yet?

**Reply by geckolinux** [reply]
@marwenius found the most definitive information here:
https://github.com/orgs/SpiralLinux/discussions/381#discussioncomment-8088229

**Reply by AlphaElwedritsch** [reply]
OK, but is it worth to use the repo?

**Reply by geckolinux** [reply]
I would say no.
